chatserver/src/main/java/pl/sages/javadevpro/Server.java

```diff
@@ -39,4 +49,25 @@ public void closeServerSocket() {
         }
     }
 
+    public void commandsHandle(String command, ChatUser user) {
```

Comment:
I don't like the method name, it is not clear for me if this is a method or variable and what it is doing. Consider changing it at some point later on.